### PR TITLE
Update contract alpha 10 (Mainnet)

### PR DESCRIPTION
**Pull request by W3stside**
Changes: contract addresses, api address (operator) and block start numbers

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr81--gpv2swap.review.gnosisdev.com)**: Testing web app

File: src/custom/state/orders/consts.ts [hunk 1]
```diff
@@ -3,6 +3,6 @@ import { ChainId } from '@uniswap/sdk'
 // TODO: fill contract deploymentblocks
 // to start checking for orders from that point
 export const ContractDeploymentBlocks: Partial<Record<ChainId, number>> = {
-  [ChainId.MAINNET]: 11424616,
+  [ChainId.MAINNET]: 11469934,
   [ChainId.RINKEBY]: 7694780
```
Comment (Velenir) [inline src/custom/state/orders/consts.ts:7]: Btw, is Rinkeby block correct?

**Review by Velenir — APPROVED** [review]

File: src/custom/state/orders/consts.ts (hunk 1 above)
Comment (W3stside) [reply src/custom/state/orders/consts.ts:7]: no idea

Comment (W3stside) [reply src/custom/state/orders/consts.ts]: updated :+1:

File: src/custom/utils/operator.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ import { registerOnWindow } from './misc'
  *    https://protocol-rinkeby.dev.gnosisdev.com/api/
  */
 const API_BASE_URL: Partial<Record<ChainId, string>> = {
-  [ChainId.MAINNET]: 'https://protocol.gnosis.io/api/v1',
+  [ChainId.MAINNET]: 'https://protocol-mainnet.dev.gnosisdev.com/api/v1',
```
Comment (anxolin) [inline src/custom/utils/operator.ts:11]: @W3stside why this change? if this is temporal. Please make sure there's a TODO and falls under your radar

Maybe this URLs should be envs? so we can change it in travis. We leave the mainnet by default, but we make travis use dev mainnet for DEV and PR branches?

Comment (W3stside) [reply src/custom/utils/operator.ts:11]: @anxolin this thread is relevant: https://gnosisinc.slack.com/archives/C6Z2XNL5Q/p1608205597200800

Comment (W3stside) [reply src/custom/utils/operator.ts:11]: but yes I'll make a note of it
